hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/snapshot/TestSnapshotRequestAndResponse.java

```diff
@@ -191,7 +192,7 @@ public void baseSetup() throws Exception {
   }
 
   @AfterEach
-  public void stop() {
+  public void stop() throws RocksDatabaseException {
```

Comment:
Instead of declaring the exception in all tests, please use `IOUtils.closeQuietly`.